packages/compartment-mapper/src/endo-types.d.ts

```diff
@@ -0,0 +1,171 @@
+// https://github.com/Agoric/SES-shim/blob/kris/endo-stack/packages/endo/DESIGN.md#compartment-maps
+
+namespace endo {
```

Comment:
Rather than a namespace, let’s perhaps give the types names that don’t collide.

Reply:
OK.

I'm lacking inspiration just now; here's hoping I find some soonish...